CMakeLists.txt

```diff
@@ -247,11 +247,13 @@ if (USE_SYSTEM_STB)
   set(MLPACK_USE_SYSTEM_STB ON)
 endif ()
 
+set(MLPACK_DONT_FIND_MLPACK 1)
```

Comment:
Just to note, this is not conditional, which means that in the case of using our own CMakeLists, we will never download the library

Reply:
Exactly, if we are configuring mlpack then we never need to actually find mlpack itself (we already have it, we are configuring it). :+1: